pkg/volume/azure_dd/azure_provision.go

```diff
@@ -143,7 +147,11 @@ func (p *azureDiskProvisioner) Provision() (*v1.PersistentVolume, error) {
 	// create disk
 	diskURI := ""
 	if kind == v1.AzureManagedDisk {
-		diskURI, err = diskController.CreateManagedDisk(name, skuName, requestGB, *(p.options.CloudTags))
+		resourceGroup := ""
```

Comment:
nit: default to c.common.resourceGroup

Reply:
I set the default in `diskController.CreateManagedDisk`, mainly it's difficult to get `c.common.resourceGroup` in this code level